distribution/changelog.txt

```diff
@@ -1,6 +1,7 @@
 0.4.20 (in development)
 ------------------------------------------------------------------------
 - Improved: [#23677] Building new ride track now inherits the colour scheme from the previous piece.
+- Fix: [#1972, #11679] Vehicles passing by toilets can cause them to glitch.
```

Comment:
It seems this also affected vanilla, so this should have " (original bug)" added to it.